pkg/xmath/stats_test.go

```diff
@@ -0,0 +1,30 @@
+package xmath_test
+
+import (
+	"testing"
+
+	"find-bin-width/pkg/xmath"
+)
+
+func TestSturges(t *testing.T) {
```

Comment:
i wonder if we shouldnt have more tests than this. finding bin widths and bins is just complicated enough, and we are just particular enough, that i worry well some day decide to change something small about our algorithm and will see unintended consequences.